Backend/src/test/java/com/onbank/api/controller/TransferControllerTest.java

```diff
@@ -0,0 +1,59 @@
+package com.onbank.api.controller;
+
+import com.onbank.AppConfig;
+import com.onbank.DatabaseConfig;
+import com.onbank.LoadProperties;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.junit.jupiter.web.SpringJUnitWebConfig;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+import org.springframework.web.context.WebApplicationContext;
+
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.print;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@ExtendWith(MockitoExtension.class)
+@LoadProperties
+@SpringBootTest
+@SpringJUnitWebConfig(classes = {AppConfig.class, DatabaseConfig.class})
+class TransferControllerTest {
+
+    private MockMvc mockMvc;
+
+    @Autowired
+    WebApplicationContext webApplicationContext;
+
+    @MockBean
+    private TransferController transferController;
```

Comment:
Nie powinniśmy wstrzykiwać kontrolera do tego testu. Po to nam jest mockMvc żeby strzelać do API i tak sprawdzać, tak jak Michał napisał. Powinny się tutaj znajdować testy podobne do void getStatusCheck(). A jeśli chcemy sprawdzić czy kontroler zwróci nam listę, to powinniśmy dodać coś do repozytorium w beforeTest. Bo w tej chwili zwróci nam pustą listę.